### DISCUSSION TITLE
The availablity for from_pyscipopt() function

### DISCUSSION THREAD

**Discussion by jp18813100494** [ds4dm/ecole#253] [Questions]
Recently, I create a new model for VRPTW problem based on Pyscipopt. Then I want to transform the Pyscipopt model to Ecole model, thus obtaining the structured observation data including column, node, edge features. However, one mistake happens when I utilized `ecole.scip.Model.from_pyscipopt()` . I wanna ask is the `from_pyscipopt()` function available now? Or I made some mistakes when I operate? Can you provide an example for `from_pyscipopt`?

**Comment by AntoinePrv** [comment]
Hi @jp18813100494,

Yes, `ecole.scip.Model.from_pyscipopt` should be working (not all cases have been tested).

> However, one mistake happens when I utilized ecole.scip.Model.from_pyscipopt()

Could you provide more information of the nature of your error, such as the error message.

> Can you provide an example for from_pyscipopt?

```python
import pyscipopt
import ecole

model_pysciopt = pyscipopt.Model()
# Keep on using the pyscipopt functions
...

# Data between the two functions are shared (same memory)
model_ecole = ecole.scip.Model.from_pyscipopt(model_pysciopt)
# Keep on using the ecole functions
...
```
